variables-skyline/index435.html

```diff
@@ -0,0 +1,197 @@
+        background:
+          linear-gradient(
+            var(--building-color1),
+            var(--window-color1)
+          );
+      }
+
+      .bb2 {
+        width: 10%;
+        height: 50%;
+      }
+
+      .bb2b {
+        width: 100%;
+        height: 100%;
+        background:
+          repeating-linear-gradient(
+            var(--building-color2),
+            var(--building-color2) 6%,
+            var(--window-color2) 6%,
+            var(--window-color2) 9%
+          );
+      }
+      
+      .bb3 {
+        width: 10%;
+        height: 55%;
+        background-color: var(--building-color3);
+      }
+
+      .bb4 {
+        width: 11%;
+        height: 58%;
+        background-color: var(--building-color4);
+      }
+
+      /* FOREGROUND BUILDINGS - "fb" stands for "foreground building" */
+      .fb1 {
+        width: 10%;
+        height: 60%;
+        background-color: var(--building-color4);
+      }
+
+      .fb2 {
+        width: 10%;
+        height: 40%;
+        background-color: var(--building-color3);
+      }
+
+      .fb3 {
+        width: 10%;
+        height: 35%;
+        background-color: var(--building-color1);
+      }
+  
+      .fb4 {
+        width: 8%;
+        height: 45%;
+        background-color: var(--building-color1);
+        position: relative;
+        left: 10%;
+      }
+      
+      .fb5 {
+        width: 10%;
+        height: 33%;
+        background-color: var(--building-color2);
+        position: relative;
+        right: 10%;
+      }
+
+      .fb6 {
+        width: 9%;
+        height: 38%;
+        background-color: var(--building-color3);
+      }
+    </style>
+  </head>
+
+  <body>
+    <div class="background-buildings">
+      <div></div>
+      <div></div>
+      <div class="bb1">
+        <div class="bb1a bb1-window"></div>
+        <div class="bb1b bb1-window"></div>
+        <div class="bb1c bb1-window"></div>
+        <div class="bb1d"></div>
+      </div>
+      <div class="bb2">
+        <div class="bb2a"></div>
+        <div class="bb2b"></div>
+      </div>
+      <div class="bb3"></div>
+      <div></div>
+      <div class="bb4"></div>
+      <div></div>
+      <div></div>
+    </div>
+
+    <div class="foreground-buildings">
+      <div></div>
+      <div></div>
+      <div class="fb1"></div>
+      <div class="fb2"></div>
+      <div></div>
+      <div class="fb3"></div>
+      <div class="fb4"></div>
+      <div class="fb5"></div>
+      <div class="fb6"></div>
+      <div></div>
+      <div></div>
+    </div>
+  </body>
+</html>
+
+<!--
+  Add these properties to `bb2a`:
+  ```
+    margin: auto;
+    width: 5vw;
+    height: 5vw;
+    border-top: 1vw solid #333;
+    border-bottom: 1vw solid #555;
+    border-left: 1vw solid #777;
+    border-right: 1vw solid #999;
+  ```
+  After you add these, you can see how a thick border on an element gives you some angles where two sides meet. You're going to use that bottom border as the top of the building.
```

Comment:
This could be broken up into several lessons. They don't need in depth descriptions, just instructing the student to add a couple of lines to `bb2a` at a time should be enough.